### DISCUSSION TITLE
Graphql list path starting with...

### DISCUSSION THREAD

**Discussion by ankyfive** [requarks/wiki#2853] [Help / Questions]
Hello,

I have the following graphql query to search and I would like to modify it to search path starting by a string : 
For instance list of all the pages that have path starting by relational...

query {
  pages {
    search(query : "relational") {
      results {
          id,
          title,
          path
      },
      suggestions,
      totalHits
    }
  }
}

Any help would be highly appraciated.
Thanks
Regards

**Comment by alexfromvl** [comment]
Use "%". list of all the pages that have path starting by "api"

``` Graphql
query {
  pages {
    search(path: "api%", locale: "en", query: "") {
      results {
        id
        title
        description
        path
        locale
      }
    }
  }
}
```
